### PR TITLE
Improve CAPI triage

### PR DESCRIPTION
**Pull request by fabriziopandini**
- Restore the triage process configuration to previous configuration
- Applies needs kind and needs priority labels to issue

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -876,16 +876,30 @@ require_matching_label:
...
   repo: cluster-api
   issues: true
   prs: false
-  regexp: ^triage/
+  regexp: ^triage/accepted$
   missing_comment: |
     This issue is currently awaiting triage.
 
-    CAPI contributors will take a look as soon as possible, apply one of the [`triage/*`](https://github.com/kubernetes-sigs/cluster-api/labels?q=triage%2F) labels and provide further guidance.
+    If CAPI contributors determines this is a relevant issue, they will accept it by applying the `triage/accepted` label and provide further guidance.
```
Comment (sbueringer) [inline config/prow/plugins.yaml:900]:
```suggestion
    If CAPI contributors determine this is a relevant issue, they will accept it by applying the `triage/accepted` label and provide further guidance.
```

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

/assign @MadhavJivrajani

**Comment by sbueringer** [comment]
*friendly reminder* @MadhavJivrajani :)

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -1696,12 +1696,6 @@ repos:
         addedBy: anyone
   kubernetes-sigs/cluster-api:
     labels:
-      - color: d455d0
-        description: Indicates an issue or PR that needs further discussion before being actively worked on.
-        name: triage/needs-discussion
-        target: both
-        prowPlugin: label
-        addedBy: org members
```
Comment (cblecker) [inline label_sync/labels.yaml:1704]: Are you wanting this label to be removed? Deleting it from here won't remove it, you need to use a `deleteAfter` field.

Comment (sbueringer) [reply label_sync/labels.yaml:1704]: We delete it manually from the repo. For CAPI that's okay

**Review by MadhavJivrajani — APPROVED** [review]
/assign @dims
for approval

**Comment by sbueringer** [comment]
Thx folks!

**Pull request merged**